lib/src/platform_app.dart

```diff
@@ -260,9 +260,9 @@ class PlatformApp extends PlatformWidgetBase<CupertinoApp, MaterialApp> {
           data?.showSemanticsDebugger ?? showSemanticsDebugger,
       debugShowCheckedModeBanner:
           data?.debugShowCheckedModeBanner ?? debugShowCheckedModeBanner,
-      theme: data?.theme,
+      theme: data?.theme?.copyWith(platform: TargetPlatform.android),
```

Comment:
Fixes the switching of platforms
```suggestion
     theme: data?.theme?.copyWith(platform: TargetPlatform.android) ??
            Theme.of(context).copyWith(platform: TargetPlatform.android),
```

Reply:
fixed, thanks